### PR TITLE
feat: add documentation for array of promises

### PR DESCRIPTION
**Pull request by brianfiszman**
The documentation that was added is about executing an array of promises secuentially

### CODE REVIEW

File: 5. Asynchronous/03. Serial Promises.md [hunk 1]
```diff
@@ -22,4 +22,21 @@ async function parallelApiCalls() {
     const [firstResult, secondResult] = await Promise.all([fetchFirst(), fetchSecond()]);
     return [firstResult, secondResult];
 }
-```
\ No newline at end of file
+```
+
+## 3️⃣ Asynchronous loop serial calls
+
+Se ejecuta de forma secuencial un array de promesas.
```
Comment (ger86) [inline 5. Asynchronous/03. Serial Promises.md:29]: Please, use english in this description. I have translated the other texts in the file

File: 5. Asynchronous/03. Serial Promises.md [hunk 2]
```diff
@@ -22,4 +22,21 @@ async function parallelApiCalls() {
...
+```
+async function serialPromiseArrayCalls(promiseArray) {
+  const results = await promiseArray.reduce(async (acc, promise) => {
+    const acumulator = await acc;
+    const res = await promise();
+
+    return [...acumulator, res];
+  }, []);
+
+  return result;
```
Comment (ger86) [inline 5. Asynchronous/03. Serial Promises.md:40]: I think the variable returned is `results`

**Comment by ger86** [comment]
Hi Brian,

I have changed some pieces of your code in order to make it work. By example, for get the result of a promise the right way is `await promise` not `await promise()`.

In order to make it clear this idea I am going to add it as a separate file because I love the idea that you mentioned in your pull request. In that new file I will mention in order to thank you your contribution.

Thank you so much!

**Comment by brianfiszman** [comment]
> Hi Brian,
> 
> I have changed some pieces of your code in order to make it work. By example, for get the result of a promise the right way is `await promise` not `await promise()`.
> 
> In order to make it clear this idea I am going to add it as a separate file because I love the idea that you mentioned in your pull request. In that new file I will mention in order to thank you your contribution.
> 
> Thank you so much!

Hi Ger. This code was tested and it works. It is `await promise()` also the way go and the `return results` is outside the reduce callback.
I guess you didn't test it. It's okay. Have fun!

**Comment by ger86** [comment]
Hi, you could view this codepen in which I use your code:

https://codepen.io/ger86/pen/wvMbrYx?editors=1111

You get this error: "Uncaught (in promise) TypeError: promise is not a function" in the console.

The correct way is `await promise` and the `result` variable has to bee `result`. 

Besides, I think is better to use a try / catch block inside reduce callback in order to capture rejected Promises.

Thanks!

**Pull request closed without merging**